modules/command/dv/dv-data-read-response-free-command.js

```diff
@@ -94,26 +94,23 @@ class DVDataReadResponseFreeCommand extends Command {
             throw errorMessage;
         }
 
-        if (privateData && Object.keys(privateData).length > 0) {
+        if (permissionedData && Object.keys(permissionedData).length > 0) {
             for (const otObject of document['@graph']) {
-                if (otObject['@id'] in privateData) {
+                if (otObject['@id'] in permissionedData) {
                     const otObjectId = otObject['@id'];
-                    for (const privateDataElement in privateData[otObjectId]) {
-                        if (!otObject.properties) {
-                            otObject.properties = {};
-                        }
-                        otObject.properties[privateDataElement] =
-                            privateData[otObjectId][privateDataElement];
+                    if (!otObject.properties) {
```

Comment:
This is duplicated code from dh-data-read-request-free-command.js it's a good example what can be placed in permissioned-data-service class